plugins/external-forward.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: external-forward
+spec:
+  homepage: https://github.com/int128/kubectl-external-forward
+  shortDescription: Connect to external host via cluster
+  description: |
```

Comment:
Please remember to wrap around at ~80 chars, so that it prints nicely on wide and narrow terminals.